examples/asr/speech_to_text.py

```diff
@@ -65,9 +64,7 @@
 
 @hydra.main(config_path="conf", config_name="config")
 def main(cfg):
-    # omegaconf merg trainer stuff to optim - this is necessary to be able to correctly setup LR scheduler
     trainer = pl.Trainer(**cfg.pl.trainer)
-    logging.info(f'Hydra config: {cfg.pretty()}')
```

Comment:
Perhaps we should log this now inside ModelPT, after creation of self._cfg?

Reply:
Not sure if we should log this at all as:
1) hydra is saving the "resulting" configuration in a separate yaml file
https://hydra.cc/docs/tutorial/working_directory
2) user has few command line args (`--h`, `--cfg` ) to display the configuration

Reply:
let's not log it in ModelPT